tensorflow/python/framework/versions.py

```diff
@@ -34,6 +35,12 @@
     "__version__",
     v1=["version.VERSION", "VERSION", "__version__"]).export_constant(
         __name__, "VERSION")
+VERSION_INFO = type('VersionInfo', (object,),__version_info__)
+tf_export(
+    "version.VERSION_INFO",
+    "__version_info__",
+    v1=["version.VERSION_INFO", "VERSION_INFO", "__version_info__"])(
+        VERSION_INFO)
```

Comment:
```suggestion
    "version.VERSION_INFO",
    "__version_info__")(
        VERSION_INFO)
```

Reply:
We don't need to export separately for v1 (in fact, v1 bindings will be removed at some point in the future if they are the same as the v2)

Reply:
Yes but it was just to be consistent in style  (see `VERSION` etc..)